src/step-function.ts

```diff
@@ -536,6 +535,45 @@ export namespace $SFN {
   });
 }
 
+function mapSDKServiceName(serviceName: string): string {
+  // source: https://docs.aws.amazon.com/step-functions/latest/dg/supported-services-awssdk.html
+  switch (serviceName) {
+    case "Discovery":
+      return "applicationdiscovery";
```

Comment:
Should we use a constant object literal instead of a switch?

Reply:
I was also thinking about that and given that we now also need a mapping for iam prefixes I would do a object literal 👍 

two questions: 
1. where should this be defined?
2. in a .ts or a .json file?

Reply:
Good question, we use `.ts` for everything but I just added an imported `.json` file in another PR haha. JSON file seems to be more appropriate for pure data and we don't lose any type safety over the data. But .ts would be more consistent with our current code. What do you think?

Reply:
I think I'll put it into the `aws.ts` file for now